saleor/dashboard/order/forms.py

```diff
@@ -144,7 +144,14 @@ def save(self):
 class ShipGroupForm(forms.ModelForm):
     class Meta:
         model = DeliveryGroup
-        fields = []
+        fields = ['shipping_method', 'tracking_number']
+
+    def __init__(self, *args, **kwargs):
+        super(ShipGroupForm, self).__init__(*args, **kwargs)
+        self.fields['shipping_method'].widget.attrs.update(
```

Comment:
If you want to expose that in the UI, you need to use the appropriate function to determine a list of allowed choices.